app/views/includes/layout.html

```diff
@@ -51,13 +51,15 @@
         <div class="govuk-header__container govuk-width-container">
             <div class="govuk-header__logo">
                 <span class="govuk-header__link--homepage">
+                  <a href="/start-eligibility" title="GOV.UK" id="logo" class="content">
```

Comment:
No need for the id and class if you're not using them.

Also, why point to /start-eligibility? It would make more sense to point to /dashboard as this page in turn would redirect to /start-eligibility if there are no applications in progress.